src/ripple/app/ledger/InboundLedgers.cpp

```diff
@@ -250,6 +257,23 @@ class InboundLedgersImp
         mLedgers.clear();
     }
 
+    std::size_t fetchRate()
+    {
+        std::lock_guard<std::mutex
+            > lock(fetchRateMutex_);
```

Comment:
To prevent the longest horizontal line from exceeding the length of the shortest horizontal line by more than the golden ratio